content/efcore/getting-started/defining-unique-identifiers.md

```diff
@@ -0,0 +1,62 @@
+# Defining Unique Identifiers 
+ 
+In this lesson we will learn about unique identifiers, how they are configured by default in EF Core, and how to manually configure them when we don't follow the convention. 
+ 
+## Primary Keys 
+ 
+When defining our data model, we need to ensure that each entity class includes a property that can be used to uniquely identify instances of that entity. These unique identifiers are referred to as **primary keys**, and they map to the concept of primary keys in a relational database.  
+ 
+By convention, EF Core assumes that a property named `Id` or `<type name>Id` is the primary key for an entity. For example, in our `Book` class below, the property named `Id` is assumed to be the primary key. 
+ 
+```{.snippet}  
+public class Book 
+{ 
+    public int Id { get; set; }  // You could also use 'BookId' here 
+    public string Title { get; set; } 
+    public string Genre { get; set; } 
+    public int PublicationYear { get; set; } 
+} 
+```  
+:::repl{data-name=primary-key}  
+::: 
+ 
+## Unconventionally Named Keys 
+ 
+In some cases, you may not want to use the conventional naming scheme for your primary key. For example, all commercially-sold books have a unique ISBN, so we could use that as a unique identifier instead of adding an extra `Id` property (assuming our library doesn't contain multiple copies of the same book). To configure an unconventionally named key like `ISBN`, use the Fluent API `HasKey` method, as shown below: 
+ 
+```{.snippet}  
+     
+public class LibraryContext : DbContext 
+{ 
+	public DbSet<Book> Books { get; set; } 
+ 
+	protected override void OnModelCreating(ModelBuilder modelBuilder) { 
+		modelBuilder.Entity<Book>() 
+			.HasKey(b => b.ISBN); 
+	} 
+} 
+ 
+```  
+:::repl{data-name=primary-key-fluent-api}  
+::: 
+ 
+## Composite Keys 
+ 
+Instead of adding a property to an entity solely for the purpose of uniquely identifying it, you may be able to use a combination of its existing properties instead. For example, we could use a book's title and publication year to uniquely identify it (assuming multiple books with the same title are not published during the same year). Combining multiple properties as a unique identifier like this is known as a **composite key**.  
```

Comment:
We say "instead" at both the beginning and end of this the first sentence. Remove one of them, please.